besu/src/main/java/org/hyperledger/besu/services/BesuPluginContextImpl.java

```diff
@@ -89,6 +90,10 @@ public void registerPlugins(final Path pluginsDir) {
       try {
         plugin.register(this);
         LOG.debug("Registered plugin of type {}.", plugin.getClass().getName());
+        final String pluginVersion = plugin.getClass().getPackage().getImplementationVersion();
```

Comment:
@rain-on should we use both `getImplementationTitle()` and `getImplementationVersion()`? That would results in  constructing version string like `events-stream 1.0.1` as opposed to adding plugin name as part of implementation version only from plugin side.

Reply:
Implementation Title does provide for friendlier names.  Perhaps impl title followed by class simple name as a fallback?

Reply:
@shemnon 👍 I was thinking on same lines. Let me get that logic in.

Reply:
method to use implementation title and fall back to class simple name is introduced.